alpha-blog/app/controllers/users_controller.rb

```diff
@@ -35,6 +38,13 @@ def update
     end
   end
 
+  def destroy
+    @user.destroy
```

Comment:
Add a `!` here. `@user.destroy!`
This is cause if there is any error which prevents this from being destroyed, we will know